src/site/layouts/health_care_region_health_services_page.drupal.liquid

```diff
@@ -8,38 +8,110 @@
+                    </div>
+                    <div class="usa-width-one-third">
+                        <a href="{{ entityUrl.path }}/become-a-patient" class="usa-button usa-button-primary">Become a patient</a>
+                    </div>
                 </div>
-                <!-- hard coded for now -->
-                <section>
-                    <p>Looking for information about shuttles, the canteen, coordinators, or other kinds of services at
-                        the Pittsburgh health care system?</p>
-                    <ul class="usa-unstyled-list">
-                        <li><a href="#">See our facility services and amenities</a></li>
-                        <li><a href="#">See our patient and family services</a></li>
+
+                <div class="usa-grid usa-grid-full vads-u-margin-top--1p5">
+                    <h3>On this page</h3>
+                    <ul>
+                        {% capture difference %} {{ careCoordinatorPatientFamilyServices.entities | size | minus:1 }} {% endcapture %}
+                        {% unless difference contains '-' %}
+                          <li><a href="#care-patient-family-services">Care Coordinators</a></li>
+                        {% endunless %}
+                        {% capture difference %} {{ socialProgramsPatientFamilyServices.entities | size | minus:1 }} {% endcapture %}
+                        {% unless difference contains '-' %}
+                            <li><a href="#social-patient-family-services">Social Programs and Services</a></li>
+                        {% endunless %}
+                        {% capture difference %} {{ healthWellnessPatientFamilyServices.entities | size | minus:1 }} {% endcapture %}
+                        {% unless difference contains '-' %}
+                            <li><a href="#wellness-patient-family-services">Health and wellness</a></li>
+                        {% endunless %}
+                        {% capture difference %} {{ primaryCareHealthServices.entities | size | minus:1 }} {% endcapture %}
+                        {% unless difference contains '-' %}
+                            <li><a href="#primary-health-services">Primary care</a></li>
+                        {% endunless %}
+                        {% capture difference %} {{ specialtyCareHealthServices.entities | size | minus:1 }} {% endcapture %}
+                        {% unless difference contains '-' %}
+                           <li><a href="#specialty-health-services">Specialty care</a></li>
+                        {% endunless %}
+                        {% capture difference %} {{ mentalHealthServices.entities | size | minus:1 }} {% endcapture %}
+                        {% unless difference contains '-' %}
+                            <li><a href="#mental-health-services">Mental health care</a></li>
+                        {% endunless %}
```

Comment:
I'm not sure where this started, but can we change these to check the array length instead of using a `capture`?  I second-guessed whether `my_array.length` worked in Liquid, but I just confirmed it does -

```
---
my_array: []
---
{% if !my_array.length %}
  <h1>empty</h1>
{% endif %}
```

☝️ Rendered "empty"